tango.py

```diff
@@ -57,7 +57,7 @@ def reset(self):
         self.center(self.SIDE)
         self.center(self.UPDOWN)
 
-    def speak(phrase):
+    def speak(self, phrase):
         ''' speak sends the given message to Android to convert to speech '''
         Client.sendMessage(phrase)
```

Comment:
Should add some kind of keyword to the front like 'speak' or something so the android server knows what to do with it